Makefile

```diff
@@ -17,6 +19,12 @@ export CRI_DOCKERD_LDFLAGS=-ldflags "-s -w -buildid=${REVISION} \
 	-X github.com/Mirantis/cri-dockerd/cmd/version.PreRelease=${PRERELEASE} \
 	-X github.com/Mirantis/cri-dockerd/cmd/version.GitCommit=${REVISION}"
 
+all: cri-dockerd
```

Comment:
I'd prefer if `all` built all of the targets and wasn't the default behavior.

Reply:
The other targets are Linux packages, which are probably not a useful default target. It likely makes more sense to make the default target `help` instead.

Reply:
But `help` doesn't build anything, making it a no-op

It breaks the "make && sudo make install" expectations

Reply:
But we can rename it

Reply:
Changed the README, the _install_ was removed anyway